UnitedIncome/components
# Input z-index shift

### Description
> Please provide a brief description of your changes below.

Really small z-index shift on the input tooltip as it's overlapping the modal right now. Nothing really should be higher than 1500. 

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Look at the `input` component. 
* Highlight it, and trigger a modal in McG.
* The input outline should not be visible over the modal. 

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
